build-tests/api-extractor-lib1-test/package.json

```diff
@@ -10,9 +10,9 @@
   },
   "devDependencies": {
     "@microsoft/api-extractor": "7.7.13",
-    "@types/jest": "23.3.11",
+    "@types/jest": "25.2.1",
     "@types/node": "10.17.13",
     "fs-extra": "~7.0.1",
-    "typescript": "~2.4.2"
```

Comment:
@sachinjoseph Do not upgrade TypeScript for this project. It is validating that API Extractor correctly analyzes the older TypeScript 2.4 syntaxes.

Reply:
Fixing now